src/systemctl/systemctl.c

```diff
@@ -4275,13 +4286,18 @@ static void print_status_info(
         }
 
         ss = streq_ptr(i->active_state, i->sub_state) ? NULL : i->sub_state;
-        if (ss)
-                printf("     Active: %s%s (%s)%s",
-                       active_on, strna(i->active_state), ss, active_off);
-        else
+        if (ss) {
+                if (!streq(i->freezer_state, "running"))
```

Comment:
This segfaults when running against an older pid1. `i->freezer_state` must be assumed to be unavailable.

Reply:
Hmm, do we support mix and match of systemctl and systemd versions?

Reply:
Yes. As mentioned on the chat, `-H` is an obvious example, but more generally, we don't require services to be immediately restarted after upgrade, so we must support version mismatches. In particular, we have no mechanism to restart `user@.service` instances, so after a `dnf ugprade`, a version mismatch may easily persist for months.

Reply:
This should be fixed in the latest revision. PTAL.